src/infra/workload/releaseunit/modules/stamp/network.tf

```diff
@@ -66,7 +62,11 @@ resource "azurerm_subnet" "kubernetes" {
   virtual_network_name = azurerm_virtual_network.stamp.name
   address_prefixes     = [module.subnet_addrs.network_cidr_blocks["kubernetes"]]
   service_endpoints = [
-    "Microsoft.Storage"
+    "Microsoft.Storage",
+    "Microsoft.AzureCosmosDB",
+    "Microsoft.KeyVault",
+    "Microsoft.ContainerRegistry",
+    "Microsoft.EventHub"
   ]
 
   enforce_private_link_endpoint_network_policies = true
```

Comment:
Based on the docs I'd assume we don't need to set this anymore.

> [enforce_private_link_endpoint_network_policies](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet#enforce_private_link_endpoint_network_policies) - (Optional) Enable or Disable network policies for the private link endpoint on the subnet. Setting this to true will Disable the policy and setting this to false will Enable the policy. Default value is false.

I'd like to drop it and give it a shot.